onnxruntime/test/python/quantization/test_get_qdq_config.py

```diff
@@ -104,6 +105,7 @@ def test_basic_args(self):
         self.assertEqual(qdq_config.activation_type, QuantType.QUInt16)
         self.assertEqual(qdq_config.weight_type, QuantType.QInt16)
         self.assertTrue(qdq_config.per_channel)
+        self.assertTrue(qdq_config.reduce_range)
```

Comment:
Note: This unit test checks that passing `reduce_range=True` to `get_qdq_config()` returns a configuration object that has the corresponding field set to True. The actual functionality of reduce_range is tested elsewhere.